tests/mtgjson5/providers/cassettes/tcgplayer_prices_map.yaml

```diff
@@ -0,0 +1,61 @@
+version: 1
```

Comment:
i guess it's a normal convention to keep the cassettes nested under the relevant tests? i wonder if it can be ./tests/cassettes ?